server/utils/email.ts

```diff
@@ -0,0 +1,69 @@
+import { Email } from "src/utils/types";
+
+const APP_BASE_URL =
```

Comment:
Instead of defining base url here let's define it this way in urls.ts so that we do not need to redefine each time we need it.